dissect/target/loaders/log.py

```diff
@@ -7,10 +10,18 @@
 
 class LogLoader(Loader):
     LOGS_DIRS = {
-        ".evtx": "sysvol/windows/system32/winevt/logs",
-        ".evt": "sysvol/windows/system32/config",
+        "evtx": "sysvol/windows/system32/winevt/logs",
+        "evt": "sysvol/windows/system32/config",
+        "iis": "sysvol/files/logs/",
     }
 
+    def __init__(self, path: Union[Path, str], **kwargs):
+        super().__init__(path)
+        uri = kwargs.get("parsed_path")
```

Comment:
If I'm correct, you can also add a optional parameter uri to the init method. If uri is in kwargs, it will be automatically filled. That way, you don't have to do the additional get ;)